arch/src/aarch64/fdt.rs

```diff
@@ -260,32 +260,51 @@ fn create_memory_node(
             fdt.end_node(memory_node)?;
```

Comment:
This commit could do with a little more explanation in the commit message.

Reply:
Ok, necessary explanations are added.